optaplanner-core/src/main/java/org/optaplanner/core/api/score/stream/bi/BiConstraintStream.java

```diff
@@ -368,6 +369,7 @@
      * Runs all tuples of the stream through a given @{@link BiConstraintCollector} and converts them into a new
      * {@link UniConstraintStream} which only has a single tuple, the result of applying {@link BiConstraintCollector}.
      * @param collector never null, the collector to perform the grouping operation with
+     * (See {@link ConstraintCollectors} for a selection of ready-made collectors.)
```

Comment:
soft suggestions:
- I'd remove the braces: "(See" -> "See"
- "for a selection of ready-made collectors.": I'd avoid the use of the word "selection" when it's not "move selection" or "entity selection" or something like that. "ready-made" is a tautology. Also, I 'd mention "count()" and "sum()" as that's much more tangible as to what I can find there (and what a collector actually is).
So suggestion => "See ConstraintCollectors for common operations, such as `count()`, `sum()` and others".